tests/src/test/java/org/jboss/weld/tests/unit/bootstrap/xml/BeansXmlTest.java

```diff
@@ -97,7 +97,7 @@ public void testInterceptorsEnabled() throws Exception {
 
             public void run(WeldManager beanManager) {
                 ModuleEnablement enabled = ((BeanManagerImpl) beanManager).getEnabled();
-                assertEquals(2, enabled.getInterceptors().size());
+                assertEquals(3, enabled.getInterceptors().size());
```

Comment:
Hm, ok. It might be better to test whether a concrete interceptor classes are present...